runtime/neumann/src/lib.rs

```diff
@@ -595,8 +589,6 @@ impl cumulus_pallet_aura_ext::Config for Runtime {}
 
 parameter_types! {
 	pub const Period: u32 = 6 * HOURS;
-	pub const Offset: u32 = 0;
```

Comment:
Looks like `Offset` was just unused? Surprised that we didn't see a compiler warning.

Reply:
ya it made me wonder what else is just unused that the compiler missed